<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Sentinel GL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="display::Widget_OpenGl" name="openGLWidget">
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_new_game"/>
    <addaction name="action_restart"/>
    <addaction name="action_similar"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="actionAbout_Sentinel"/>
    <addaction name="actionAbout_remake"/>
    <addaction name="actionAbout_mhk"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionGPL"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_new_game">
   <property name="text">
    <string>&amp;Game setup ...</string>
   </property>
  </action>
  <action name="actionAbout_Sentinel">
   <property name="text">
    <string>About the Sentinel ...</string>
   </property>
  </action>
  <action name="actionRules">
   <property name="text">
    <string>About the rules ...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt ...</string>
   </property>
  </action>
  <action name="actionAbout_openGL">
   <property name="text">
    <string>About openGL ...</string>
   </property>
  </action>
  <action name="actionAbout_mhk">
   <property name="text">
    <string>About me ...</string>
   </property>
  </action>
  <action name="actionTest1">
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionTest2">
   <property name="text">
    <string>Test2</string>
   </property>
  </action>
  <action name="action_opengl_setup">
   <property name="text">
    <string>openGL setup ...</string>
   </property>
  </action>
  <action name="action_restart">
   <property name="text">
    <string>Restart current level</string>
   </property>
  </action>
  <action name="action_similar">
   <property name="text">
    <string>Start similar level</string>
   </property>
  </action>
  <action name="actionAbout_remake">
   <property name="text">
    <string>About this remake ...</string>
   </property>
  </action>
  <action name="actionGPL">
   <property name="text">
    <string>Gnu Public License ...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>display::Widget_OpenGl</class>
   <extends>QOpenGLWidget</extends>
   <header>widget_openGl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
